Automattic/pocket-casts-ios
# Playback 2024: Listening Time Story + sharing refactors

| 📘 Part of: #2250 |
|:---:|

Completes #2325

| | |
| -- | -- |
| <img src="https://github.com/user-attachments/assets/6923104d-80be-4f6d-89f3-16fceb6f5989" width=300> | <img src="https://github.com/user-attachments/assets/393ef230-ee85-4dba-9895-26bae472c057" width=300> |

## To test

* Enable the `endOfYear2024` feature flag
* Restart the app
* Navigate to the Playback 2024 cell in Profile
* Navigate through stories until you reach this one
* Ensure stats show correctly

## Checklist
